tokio-trace-fmt/src/lib.rs

```diff
@@ -238,7 +241,12 @@ impl<N, E, F> Builder<N, E, F> {
 
     /// Sets the subscriber being built to use the default full span formatter.
     // TODO: this should probably just become the default.
-    pub fn full(self) -> Builder<N, fn(&span::Context, &mut fmt::Write, &Event) -> fmt::Result, F> {
+    pub fn full(
+        self,
+    ) -> Builder<N, fn(&span::Context<N>, &mut fmt::Write, &Event) -> fmt::Result, F>
+    where
+        N: for<'a> NewVisitor<'a> + 'static,
```

Comment:
I don't _think_ this bound is necessary, since the impl block already has it.

Reply:
the impl block does not have it only this fn and with_visitor have the bound for N

Reply:
Sorry, you're right --- I misread the diff. Nevermind!